app/views/cases/overturned_sar/_new_form.html.slim

```diff
@@ -36,12 +24,13 @@ hr
   = f.gov_uk_date_field :external_deadline, { legend_text: "Final deadline" ,
           form_hint_text: "We'll use this date to set the draft deadline, calculated as 20 working days from the final deadline."}
 
+
 p
   = "Only change the details below if the Requester's contact details and the "
   = "preferred way of receiving the information has changed."
 
 #delivery-method
-  = f.radio_button_fieldset :delivery_method, choices: [:send_by_email, :send_by_post] do |fieldset|
+  = f.radio_button_fieldset :reply_method, choices: [:send_by_email, :send_by_post] do |fieldset|
```

Comment:
Can we sync up the id of the div here with the field name?

@aliuk2012 Do you reckon we should just remove it IF it's not even being used?

Reply:
@misha I've got a feeling its used for tests but I agree with you it should be sync up to the field that we are using.  BTW why are we using a new attr called reply_method? Isn't this the same information we are asking for on regular SARs couldn't we just reuse the same attributes as a regular SAR i.e delivery_method. i